cmd/pg/wal_restore.go

```diff
@@ -9,23 +9,64 @@ import (
 
 const (
 	WalRestoreUsage            = "wal-restore target-pgdata source-pgdata"
-	WalRestoreShortDescription = "Restores WAL segments from storage."
-	WalRestoreLongDescription  = "Restores the missing WAL segments that will be needed to perform pg_rewind from storage."
+	WalRestoreShortDescription = "Restores WAL segments from storage to the source to perform pg_rewind with the target."
+	WalRestoreLongDescription  = "Restores the missing WAL segments that will be needed in source cluster to perform pg_rewind " +
+		"with target cluster from storage. If you mark the target cluster as remote you should specify requisites to connect by" +
```

Comment:
Add the note that in the case of remote mode, `wal-restore`  should be run from the `source` cluster.

Reply:
Added.